src/item_location.cpp

```diff
@@ -566,6 +661,14 @@ void item_location::deserialize( JsonIn &js )
     }
 }
 
+item_location item_location::parent_item() const
+{
+    if( where() == type::container ) {
+        return ptr->parent_item();
+    }
+    return *this;
```

Comment:
This seems counter intuitive to me, I think you should at least mention in the comment documenting this function that if an item is not in a container, it is its own parent.  Is this behavior relied on for the current code?

Reply:
do you think it should be nullptr instead?

Reply:
I think it should return a nowhere item_location and trigger a debug message